tests/components/vacuum/test_intent.py

```diff
@@ -37,6 +37,27 @@ async def test_start_vacuum_intent(hass: HomeAssistant) -> None:
     assert call.data == {"entity_id": entity_id}
 
 
+async def test_start_vacuum_without_name(hass: HomeAssistant) -> None:
```

Comment:
I feel like some pytest.parametrize might be cool, as the four tests are basically the same with a few minor adjustments

Reply:
100%, but that is fine to do in a follow up as well 👍